.github/workflows/test-workflow.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Frontend and backend
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-frontend:
+    name: Test ember frontend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository files
+        uses: actions/checkout@v2
+      - name: Install node js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install node dependencies
+        run: yarn install
+      - name: Run frontend tests
+        run: yarn test
```

Comment:
This seems to miss some tests:
- linting for JavaScript files: `yarn lint:js`
- linting for Templates: `yarn lint:hbs`
- tests against BrowserStack: `node_modules/ember-cli/bin/ember browserstack:connect && yarn test --config-file testem.browserstack.js && node_modules/ember-cli/bin/ember browserstack:disconnect`
- bundle size tests: ```yarn test:bundlesize```
- test that CSP in `.htaccess` is up-to-date: ```grep "`node_modules/ember-cli/bin/ember csp-headers --environment production --silent 2>&1 | sed 's/ $//'`" public/.htaccess || (echo "CSP headers in public/.htaccess does not match configuration" && exit 1)```

We might want to provide aliases in `package.json` for the BrowserStack and CSP in `.htaccess` tests. The BrowserStack tests require credentials. You could find them in old `.travis.yml`.

Reply:
ok I will add this tests